src/chrome/content/rules/iliad.it.xml

```diff
@@ -0,0 +1,22 @@
+<!--
+All other subdomains are either internal or test pages (they give 
+403 error or SSL_ERROR_BAD_CERT_DOMAIN error because of their test 
+certificate).
+-->
```

Comment:
Please indent the comment's contents:
```xml
<!--
	All other subdomains are either internal or test pages (they give 
	403 error or SSL_ERROR_BAD_CERT_DOMAIN error because of their test 
	certificate).
-->
```